modules/likes.php

```diff
@@ -557,24 +557,46 @@ function admin_bar_likes() {
 	}
 }
 
+/**
+ * Callback to get the value for the jetpack_likes_enabled field.
+ */
+function jetpack_post_likes_get_value( array $post ) {
+	$post_likes_switched = get_post_meta( $post['id'], 'switch_like_status', true );
+
+	$sitewide_likes_enabled = (bool) apply_filters( 'wpl_is_enabled_sitewide', ! get_option( 'disabled_likes' ) );
```

Comment:
Could you add this above the filter here and below, to avoid tripping our parser?
`/** This filter is documented in modules/likes.php */`

Reply:
Done. FWIW I don't see that it actually *is* documented (though I do see other comments saying it is).

Reply:
Ouch, that's a good point. The filter got moved to `modules/likes/jetpack-likes-settings.php` a while ago, but the references never got updated.

Reply:
I was back in here anyway so I updated the references in this particular file.